package.json

```diff
@@ -29,25 +29,28 @@
   ],
   "license": "MIT",
   "engines": {
-    "node": ">= 6.0.0"
+    "node": ">= 8.0.0"
   },
   "dependencies": {
+    "@types/node": "^8.10.0",
```

Comment:
Should `@types/node` and `@types/pg` be `devDependencies`?

Reply:
Need to understand how to do this right. I thought I had trouble with installation in a consuming repo if the type wasn’t already included.